### DISCUSSION TITLE
If restart reason isn't power on then do nothing?

### DISCUSSION THREAD

**Discussion by javier6d2** [arendst/Tasmota#18219] [Q&A]
Hi, I have a dimmer device that randomly reboots.
It would be possible to create a if statement with rules or scripting like:

If restart reason isn't power on then do nothing?

Kind regards

**Comment by javier6d2** [comment]
Rule1 
ON ?#RestartReason $! Power On DO xxxxxxx ENDON

**Reply by barbudor** [reply]
It's difficult to say "do nothing" without knowing what it actually does that bugs you
